package.json

```diff
@@ -126,6 +126,7 @@
     "sparqljs": "^3.4.2",
     "url-join": "^4.0.1",
     "uuid": "^8.3.2",
+    "verror": "^1.10.1",
```

Comment:
@joachimvh Maybe that's good to have as a future todo; wouldn't do it here yet.

Reply:
veror is pretty lightweight dependency. And is from joyent itself, (one of core maintainers of node)

Reply:
That's all fine; it's just that we need to be consistent across our codebase.

Reply:
git that. Though, it is yet another subclass of `Error`, just like many Error types in `CSS`. Instead of implementing ourselves, we are importing one, which provides chained error stack.

Reply:
Exactly; so we should propagate that to the rest of CSS in a separate PR. But just adding it in one place here introduces inconsistencies.

I'd propose more lightweight error reporting here, that would be fine for now.